apps/web/src/ee/clerk/providers/ClerkProvider.tsx

```diff
@@ -16,13 +16,166 @@ const CLERK_LOCALIZATION = {
       security: 'Access security',
     },
   },
-
+  organizationProfile: {
+    membersPage: {
+      requestsTab: { autoSuggestions: { headerTitle: '' } },
+      invitationsTab: { autoInvitations: { headerTitle: '' } },
+    },
```

Comment:
This is done to hide links that break the flow of modals. The functionality is exposed in a dedicated space in a specific settings tab so it is just about hiding a small element on the UI that can cause the modal to break